public/components/query_compare/search_result/index.tsx

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  EuiPage,
+  EuiPageBody,
+  EuiPageHeader,
+  EuiTitle,
+  EuiPageContentBody,
+  EuiPageHeaderSection,
+  EuiCode,
+  EuiPanel,
+  EuiSpacer,
+  EuiFlexItem,
+  EuiButtonIcon,
+  EuiFlexGroup,
+  EuiSplitPanel,
+  EuiText,
+} from '@elastic/eui';
+import React, { useEffect, useState } from 'react';
+
+import { CoreStart, ChromeBreadcrumb } from '../../../../../../src/core/public';
+import { SearchConfigsPanel } from './search_components/search_configs';
+import { SearchInputBar } from './search_components/search_bar';
+import { SearchResultTable } from './result_components/search_result_table';
+import { ServiceEndpoints } from '../../../../common';
+
+const defaultQuery = JSON.stringify({
+  query: {
+    term: { title: '%SearchQuery%' },
+  },
+});
+
+interface SearchResultProps {
+  http: CoreStart['http'];
+}
+
+export const SearchResult = ({ http }: SearchResultProps) => {
+  const [isCollapsed, setIsCollapsed] = useState(false);
+  const [searchIndex1, setSearchIndex1] = useState('');
+  const [searchIndex2, setSearchIndex2] = useState('');
+  const [queryString1, setQueryString1] = useState('');
+  const [queryString2, setQueryString2] = useState('');
+  const [querqyResult1, setQuerqyResult1] = useState({});
+  const [querqyResult2, setQuerqyResult2] = useState({});
```

Comment:
Can we find/replace all occurrences of querqyResult with queryResult?

Given that there is a thing called Querqy, this apparent typo is confusing.